internal/devices/drivers.go

```diff
@@ -73,19 +73,24 @@ func execModprobeInstallDriver(ctx context.Context, vm *uvm.UtilityVM, driverDir
 		Stderr: p,
 	}
 
-	exitCode, err := cmd.ExecInUvm(ctx, vm, req)
-	if err != nil && err != noExecOutputErr {
-		return errors.Wrapf(err, "failed to install driver %s in uvm with exit code %d", driverDir, exitCode)
-	}
+	var (
+		execErr  error
+		exitCode int
+	)
+	exitCode, execErr = cmd.ExecInUvm(ctx, vm, req)
```

Comment:
Couldn't this be `exitCode, execErr := cmd.ExecInUvm(ctx, vm, req)`?

Reply:
yeah I can clean that up